### ISSUE TITLE
Swift 3 and CocoaPods

### ISSUE THREAD

**Issue by diek85** [whitesmith/WSTagsField#23]
Hi

I just tried to install your library via CocoaPods, but it seems that it gets the 2.3 Swift version of the code. 

My pod files simply contains 

`pod "WSTagsField"`

What should I do?

**Comment by ricardopereira** [comment]
@diek85 Strange. https://cocoapods.org/pods/WSTagsField is linking to the latest https://github.com/whitesmith/WSTagsField/releases/tag/2.0.0 so it should be working. Can you try it with: `pod 'WSTagsField' '~> 2.0.0'`?

**Comment by diegovigano** [comment]
Hi,

Actually it was retrieving the 1.2.2 version. 

<img width="566" alt="schermata 2016-11-11 alle 08 58 39" src="https://cloud.githubusercontent.com/assets/13921451/20208389/2e3558fc-a7ee-11e6-99b3-9957d2383800.png">

Adding 2.0.0 resulted in a CocoaPods error (saying that `None of your spec sources contain a spec satisfying the dependencies`).

So I had to perform a `pod repo update` and after that I was able to get 2.0.0 version and not it works like a charm.

Thanks for your support :)
